docs/components/text/README.md

```diff
@@ -0,0 +1,1288 @@
+    </cdr-text> 
+    “10 Best Pants for Working Outside” 
+  </cdr-text>
+
+```
+</cdr-doc-example-code-pair>
+
+<cdr-doc-example-code-pair repository-href="/src/components/text" :sandbox-data="$page.frontmatter.sandboxData">
+
+```html
+
+  <cdr-text modifier="body-300">
+    This trip is rated as
+    <cdr-text
+    tag="strong"
+    modifier="strong">
+    Vigorous [4]
+    </cdr-text>.
+  </cdr-text>
+
+```
+</cdr-doc-example-code-pair>
+
+## Accessibility
+
+### Headings
+
+To ensure that usage of this component complies with accessibility guidelines:
+  - Use h1-h6 to identify headings (`<h1>`, `<h2>`, `<h3>`, `<h4>`, `<h5>`, and `<h6>`)
+  - Headings are used to label page regions
+    - Use the aria-label attribute to associate headings with their page region, as described in the [label page regions](https://www.w3.org/WAI/tutorials/page-structure/labels/#using-aria-labelledby) tutorial
+  - [Subheadings](#subheadings) are not semantic headings. Subheadings may be visually styled as a heading, but will not be navigable using a screen reader
+  - Assistive technologies skim the structure of a page:
+    - Allow users to navigate to or skip over sections through the use of heading levels
+    - Avoid skipping heading levels (e.g., `<h2>` to  `<h4>` )
+
+This component has compliance with WCAG guidelines by:
+  - Defining semantic heading levels with the ability to assign predefined visual heading styles to each level
+
+### Paragraphs
+
+To ensure that usage of this component complies with accessibility guidelines:
+
+  - Text container does not exceed 634px or line length does not exceed more than 75 characters
+  - Minimize use of reversed-out body copy (light copy on a dark background) because it’s harder to read
+  - When possible, use the first sentence as an introduction to the paragraph. With screen readers, users can listen to the first sentence and then jump to the next paragraph
+  - Break long pages into shorter sections by organizing content into well-defined groups or chunks
+
+This component has compliance with following WebAIM’s accessibility guidelines:
+
+  - [WCAG SC 1.4.8: Visual presentation](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-visual-presentation): Cedar Design System text component provides for spacing for:
+    - Within paragraphs, line spacing is at least 1.5 times font height
+
+# Implementation: 
+
+ To work as intended, Tokens, Utility classes and Component require both `cdr-reset.css` and access to the REI Brand fonts. for more information on how to access these resources visit the [developer getting started](https://rei.github.io/rei-cedar-docs/getting-started/as-a-developer/?active-link=setting-up-projects) page.
+
+ The global text default `cdr-text-utility-300` is defined in each of the available cdr-container classes. Make sure that all content is contained in one of the [available container classes](http://localhost:8080/layout/responsive/#standard-vs-fluid-container)
```

Comment:
the reset also puts `cdr-text-default` on `body`, not sure how that plays into this?